src/users/users.routing.ts

```diff
@@ -1,7 +1,7 @@
 // routes
 import { ModuleWithProviders } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
-import { HealthCheckGuard } from 'src/shared/guards/health-check.guard';
+import { HealthCheckGuard } from '../../src/shared/guards/health-check.guard';
```

Comment:
@SivaprakasamSivaprakasam - are you sure this import is correct? Usually you wouldn't expect to see `src` anywhere. Is it not `../shared/guards/health-check.guard`?